recipe/meta.yaml

```diff
@@ -10,16 +10,15 @@ source:
   sha256: 863b820f4ddd0715d01ae7c38047b9396d35fb705222bbaeae5f59e708c0cfa0
 
 build:
-  noarch: python
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 1
+  merge_build_host: true  # workaround conda-build bug on windows
```

Comment:
What error message are you getting?

Reply:
See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=686673&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=1183ba29-a0b5-5324-8463-2a49ace9e213&l=324. Last time I observed this, this was the key which fixed it (and it did again).